source/help/settings/channel-settings.rst

```diff
@@ -0,0 +1,59 @@
+Channel Settings
+================
+
+--------------
+
+Notification preferences, channel header, channel purpose, and channel
+name are customizable for each channel. To access these settings, click
+the channel name at the top of the page to open the channel drop-down
+menu.
+
+Channel Notification Preferences
+--------------------------------
+
+Notification preferences can be modified for each channel you belong to.
+
+Send Desktop Notifications
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, the desktop notification preference assigned in *Account
+Settings* is used for all channels. To customize the desktop
+notification preference for each channel, click the channel name at the
+top of the page to access the drop-down menu, then click
+**Notification Preferences > Send Desktop Notifications**.
+
+Mark Channel Unread
+~~~~~~~~~~~~~~~~~~~
+
+By default, channel names are bolded for all new messages in a channel. 
+To only bold the channel name when you are mentioned, open the channel drop-down menu and click 
+**Notification Preferences > Mark Channel Unread > Only for mention**.
+
+
+Channel Header
+--------------
+
+Select **Edit Channel Header** to change the text that appears next to
+the channel name at the top of the screen. It can be used to summarize
+the channel topic or provide links to frequently accessed documents. Any
+channel member can edit this setting, unless the System Administrator
```

Comment:
Do we really need to change this to System Admin? 

If so it might be best to make a separate issue for this, as I think there are many places in the docs and in the product help text that use Administrator.